### PR TITLE
Implelemt Field/{field_id} resource PUT (Feature/#128)

### LINKED ISSUE #128: Implement Field/{id} resource PUT

### PR DESCRIPTION
**Pull request by zzeropilot**
- updated field service
- updated field route

### CODE REVIEW

File: src/app/routers/field.py [hunk 1]
```diff
@@ -215,3 +224,101 @@ def get(self, field_id):
...
+                range_max=range_max,
+                range_min=range_min,
+                added_choice_options=added_choice_options,
+                removed_choice_options=removed_choice_options
+            )
+
+        if response is None:
+            raise BadRequest("Couldn't update")
+
+        return Response(status=204)
```
Comment (IvanKostiv) [inline src/app/routers/field.py]: status=200

File: src/app/routers/field.py [hunk 2]
```diff
@@ -215,3 +224,101 @@ def get(self, field_id):
                 field_json[key] = value
 
         return jsonify(field_json)
+
+    @API.doc(
+        responses={
+            200: "OK",
+            204: "No Content",
```
Comment (IvanKostiv) [inline src/app/routers/field.py]: delete 'No Content"

File: src/app/services/field.py [hunk 3]
```diff
@@ -555,3 +568,183 @@ def check_for_range(data):
...
+        :param sheet:
+        :param from_row:
+        :param to_row:
+        :return:
+        """
+
+        field = FieldService.update(field_id=field_id, name=name)
+        settings = SettingAutocompleteService.get_by_field_id(field_id)
+        if settings is None:
+            raise FieldNotExist()
```
Comment (IvanKostiv) [inline src/app/services/field.py]: SettingAutocompleteNotExist()

File: src/app/routers/field.py (hunk 1 above)
Comment (zzeropilot) [reply src/app/routers/field.py]: fixed

File: src/app/routers/field.py (hunk 2 above)
Comment (zzeropilot) [reply src/app/routers/field.py]: fixed

File: src/app/services/field.py (hunk 3 above)
Comment (zzeropilot) [reply src/app/services/field.py]: fixed

**Review by yurdosii — DISMISSED** [review]

**Review by AndriiKret — DISMISSED** [review]

**Review by IvanKostiv — APPROVED** [review]

**Review by yurdosii — APPROVED** [review]

**Review by AndriiKret — APPROVED** [review]

**Pull request merged**